examples/classifier/CL1_compute_dtw_distance.py

```diff
@@ -0,0 +1,33 @@
+# Standard library imports
+import sys
+new_path = sys.path[0][:-len("/examples/timeseries")]
+sys.path.append(new_path)
```

Comment:
why sys here? for setting correct path for import?

I would prefer using absolute path for import, e.g. `scifin.timeseries.randomseries import auto_regressive`

Reply:
Thanks. This is a remaining from older implementation. I forgot to change it. Will correct it right now.